src/products/components/ProductVariantPrice/ProductVariantPrice.tsx

```diff
@@ -123,6 +124,9 @@ const ProductVariantPrice: React.FC<ProductVariantPriceProps> = props => {
                 const error = formErrors.price?.channels?.find(
```

Comment:
Since costPriceError has a defining name, maybe we could also change error to priceError or so to match?